spec/features/profiles_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+feature "zarządzanie profilem użytkownika" do
+  context "po zalogowaniu" do
```

Comment:
Tutaj brakuje kodu do logowania się. Jednak żeby to miało sens, trzeba zalogować się jako "Anna Genialna".